samples/glean/metrics.yaml

```diff
@@ -34,9 +34,7 @@ test:
   test_string_list:
     type: string_list
     description: >
-      Testing StringList ping
-    send_in_pings:
-      - test_string_list
```

Comment:
Why did you change this? Doesn't this make the test metric available in all pings?

Reply:
I did this to get more data in the default or metrics ping for testing.